### PR TITLE
Issue #5, Add Item class and alter CMake 

### PR DESCRIPTION
**Pull request by Jahrr**
This pull request adds the Item class, which will serve as a starting point for sudoManager. This contains the variables Item ID, name, price, stock, and additional info for the item. When finding an object, itemID should always be used, rather than the name. Item IDs should also be exclusive to a specific item. CMake was altered to include the include directory, add add the source file variable. Resolves Issue #5

### CODE REVIEW

**Comment by Jahrr** [comment]
CMake needs to be edited before merging

**Comment by Jahrr** [comment]
This works, but I had to make the sudoManager library static. My CMake knowledge is limited, so I didn't know how to make it work with a shared library, and i didnt want to set link_directories to the build folder. If we could specify a lib folder to build the libraries to and then link to that, it would be better.

File: src/Item.cpp [hunk 1]
```diff
@@ -0,0 +1,52 @@
+//
+// Created by Jahrr on 4/6/2023.
+//
+// Last edited by Jahrr on 4/6/2023
+//
+#include "../include/sudoManager/Item.hpp"
```
Comment (genciberisha) [inline src/Item.cpp:6]:
```suggestion
#include "sudoManager/Item.hpp"
```
No need for a relative path.
`/src/CmakeLists.txt` will handle that

File: extern/CMakeLists.txt [hunk 2]
```diff
@@ -1 +0,0 @@
-add_subdirectory(googletest)
```
Comment (genciberisha) [inline extern/CMakeLists.txt:1]: Removing this will break the Gtest suite build.
Revert this.

**Review by genciberisha — CHANGES_REQUESTED** [review]
Thank you for your contribution!
Resolve the comments and after that squash all commits into one.

**Review by RockyDesigne — APPROVED** [review]

**Review by kelarov — COMMENTED** [review]
Reviewed

**Review by genciberisha — APPROVED** [review]

**Pull request merged**